test/libsolidity/util/BytesUtils.cpp

```diff
@@ -263,8 +263,15 @@ string BytesUtils::formatBytes(
 		os << formatHexString(_bytes);
 		break;
 	case ABIType::String:
-		os << formatString(_bytes);
+	{
+		auto paddingZerosCnt = find_if(
```

Comment:
I think it would be better to move this into its own function and then call it from here.

Reply:
Maybe `BytesUtils::countRightPaddedZeros`?

Reply:
Done that here 8a8f553e4d1ada792bb31f1b15df3675f4cc7f64